src/hex_utils.c

```diff
@@ -14,50 +14,50 @@ uint8_t hex_parseNibble(const char c)
 	THROW(ERR_UNEXPECTED_TOKEN);
 }
 
-uint8_t hex_parseNibblePair(const char* str)
+uint8_t hex_parseNibblePair(const char* inStr)
```

Comment:
This is not `Str` as we do not expect it to be null-terminated! I guess we might also add `inSize` as well and check it is at least 2